Header header

bool power
float32 x_translation
float32 y_translation
float32 lift
float32 angle
bool joystickControl
